### PR TITLE
Service Fabric 8.0 CU1 Release Notes

### PR DESCRIPTION
**Pull request by sukanyamsft**
Service Fabric 8.0 CU1 Release Notes

### CODE REVIEW

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Service Fabric SDK will now support Startup Services as an alternative to Default Services specified in Application Manifest
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80CU1.md:29]: Do we have documentation on how to use Startup Services?

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+|-|-|-|-|
+| **Windows - 8.0.516.9590<br>Ubuntu 16 - 8.0.515.1<br>Ubuntu 18 - 8.0.515.1804** | **Feature** | Service Fabric SDK will now support Startup Services as an alternative to Default Services specified in Application Manifest | **Brief desc**: Service Fabric SDK that will support Startup Services as an alternative to Default Services specified in Application Manifest for the Visual Studio debugging experience.<br>**Impact**: This is an additional feature and will not have any impact on existing behavior.<br>**Change**: This features provides support for StartupServices.xml as an alternative to Default Services specified in ApplicationManifest.xml for the Visual Studio debugging experience. With this implementation, DefaultServices and Service definition related parameters are moved from existing ApplicationManifest.xml to a new file called StartupServices.xml. Functionality for each function - from addition of a new service to deployment (Build/Rebuild/F5/Ctrl+F5/Publish experiences in Visual Studio) is provided to work this new feature end to end. For end user to use this, VS tooling team should release the corresponding feature in VS as well. |
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80CU1.md:60]: Can we add some insight as to why a customer would want to switch to startup services?

Comment (JohnNilsson) [reply release_notes/Service_Fabric_ReleaseNotes_80CU1.md:60]: There has been some communication on this throughout the years: https://stackoverflow.com/questions/50441340/auto-creation-of-service-without-defaultservices-on-developer-machines/50445801#50445801

I would also be interested in an official documentation on how to safely dig oneself out of the hole of having default services in a production app towards a "best practice" setup. (f.ex. I think there's some setting that must be set to not accidentally delete services)

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md (hunk 1 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80CU1.md:29]: As discussed, removed note about startup services.

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md (hunk 2 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80CU1.md:60]: As discussed, removed note about startup services.

**Review by peterpogorski — APPROVED** [review]

**Comment by sukanyamsft** [comment]
#signoff

**Pull request merged**